mods/voidmulti.sc2mod/base.sc2data/gamedata/unitdata.xml

```diff
@@ -30,6 +32,7 @@
         <GlossaryStrongArray value="Mutalisk"/>
         <GlossaryStrongArray value="Adept"/>
         <GlossaryWeakArray index="1" value="Hydralisk"/>
+        <InnerRadius value="0.5625"/>
```

Comment:
Archon can now fit between single-gap walls. (Collision size with other units unaffected)
- InnerRadius reduced from 0.75 to 0.5625